content/docs/deployment/helm/drivers/installation/powermax.md

```diff
@@ -273,12 +273,38 @@ CRDs should be configured during replication prepare stage with repctl as descri
     ```bash
     kubectl create -f samples/secret/secret.yaml
     ```
-5. Download the default values.yaml file
+5. Create a configmap using below sample file. Fill in the appropriate values for driver configuration. Example: X_CSI_TRANSPORT_PROTOCOL:"ISCSI" 
+   ```yaml  
+      # Copyright © 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+      #
+      # Licensed under the Apache License, Version 2.0 (the "License");
+      # you may not use this file except in compliance with the License.
+      # You may obtain a copy of the License at
+      #      http://www.apache.org/licenses/LICENSE-2.0
+      # Unless required by applicable law or agreed to in writing, software
+      # distributed under the License is distributed on an "AS IS" BASIS,
+      # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      # See the License for the specific language governing permissions and
+      # limitations under the License.
+      # To create this configmap use: kubectl create -f powermax-array-config.yaml
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: powermax-array-config
+        namespace: powermax
+      data:
+        powermax-array-config.yaml: |
+          X_CSI_POWERMAX_PORTGROUPS: "" # Portgroup is required in case of iSCSI only
+          X_CSI_TRANSPORT_PROTOCOL: "" # Defaults to empty
+          X_CSI_POWERMAX_ENDPOINT: "https://10.0.0.0:8443"
+          X_CSI_MANAGED_ARRAYS: "000000000000,000000000000,"
```

Comment:
I'm not sure if it matters, but all fields here besides the first are grayed out in the Changes tab. 
It may be related to the `|` on the .yaml line. Is that pipe necessary?